providers.cpp

```diff
@@ -796,23 +796,30 @@ bool VirtualBoxProvider::isOutDated() const
 class MathItem::Parser : public mu::Parser
 {
 public:
-    static void initLocale()
+    Parser()
     {
         try
         {
             // use the system's locale instead of the "C"
-            s_locale = std::locale{""};
+            const std::locale loc{""};
+            auto && numpunct = std::use_facet<std::numpunct<mu::char_type>>(loc);
+            SetDecSep(numpunct.decimal_point());
+
+            const char arg_sep = GetArgSep();
+            if (numpunct.decimal_point() == arg_sep)
+            {
+                if (arg_sep == ',')
+                    SetArgSep(';');
+                else
```

Comment:
hmm - we only have to act if `numpunct.decimal_point() == arg_sep` is true. To act consistent i would prefer not to check and set SetArgSep(';') anyways. Reason: 
* de and others: 1.000.000,00 - ups, we should change the arguments separator
* en and others: 1,000,000.00 - ups, the don't check the thousands separator - maybe they will do later

This is pure academical, if you don't agree i'm fine with. In that case GTM